### ISSUE TITLE
Can this model reimplement the training procedure of PSPNet(hszhao) ?

### ISSUE THREAD

**Issue by zchengquan** [hellochick/PSPNet-tensorflow#11]
Hi, can i use this code to reimplement the training step of PSPNet, and obtain  comparable results reported in the paper of PSPNet?

**Comment by hellochick** [comment]
It's hard to reimplement the training step of PSPNet because of the batch size should be large enough to update mean/var. The original author use openMP or MPI to update those variable in bn layer.

**Comment by John1231983** [comment]
@hellochick : I have an idea for small batch size. We can use accumulate gradient as the Deeplab did. Do you think it will solve the problem?

**Comment by FengLoveBella** [comment]
It is a GPU-consuming network. The batch size can only be set to 1 when I try to run the train code.
